tools/clang/unittests/HLSL/TestMain.cpp

```diff
@@ -142,6 +140,8 @@ int main(int argc, char **argv) {
 #endif
 #endif
 
+  moduleSetup();
   int rv = RUN_ALL_TESTS();
+  moduleTeardown();
```

Comment:
The fact that this is done here is largely historical. Previously, these functions were implemented in VerifierTest.cpp.